### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by Ducica**

### CODE REVIEW

File: oarepo_ui/templates/oarepo_ui/header_login.html [hunk 1]
```diff
@@ -21,6 +21,47 @@
             {% endif %}
         </form>
   {%- else -%}
+
+      {# Desktop Quick-create ("plus") menu #}
+    {%- set plus_menu_items = current_menu.submenu('plus').children %}
+    {%- if plus_menu_items %}
+      <div
+        id="quick-create-dropdown"
+        class="rdm-plus-menu rdm-plus-menu-responsive ui dropdown pr-15 computer only"
```
Comment (mirekys) [inline oarepo_ui/templates/oarepo_ui/header_login.html:30]: Do we have these rdm* classes defined in oarepo-ui?

Comment (Ducica) [reply oarepo_ui/templates/oarepo_ui/header_login.html:30]: @mirekys We don't, you are right, I will remove it (it is something used in RDM). Sorry!

File: oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_footer.less [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+@import (multiple) "../../theme.config";
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+ */
+
+.ui.footer-component {
```
Comment (mirekys) [inline oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_footer.less:37]: Class name should be the same as component name - `.ui.page-footer`

File: oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_header.less [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+@import (multiple) "../../theme.config";
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+ */
+
+.ui.header-component {
```
Comment (mirekys) [inline oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_header.less:37]: Class name should be the same as component name - .ui.page-header

File: oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_header.less [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+        background-color: @pageHeaderMainNavLinkLargeScreen !important;
+      }
+    }
+  }
+
+  #invenio-nav.ui.menu {
+    height: @relative75px;
+    position: relative;
+
+    .ui.menu {
```
Comment (mirekys) [inline oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/definitions/page_header.less:84]: Is selector for `.ui.menu` needed here? Is there such nested menu component under `#invenio-nav`? Maybe `#invenio-menu` was meant here? I would use id selector then :)

File: oarepo_ui/templates/components/Searchbar.jinja [hunk 5]
```diff
@@ -0,0 +1,19 @@
+{# def search_link="/search"  #}
+{% set options = [
+    {
+      "key": "docs",
```
Comment (mirekys) [inline oarepo_ui/templates/components/Searchbar.jinja:4]: Remove references to NR-docs, keep it just as "All records", or, even better, let devs pass the whole options array as Searchbar component prop with a default value of "All records" on `/search` endpoint

**Review by mirekys — CHANGES_REQUESTED** [review]

**Review by mirekys — APPROVED** [review]

**Review by mirekys — APPROVED** [review]

**Pull request merged**